wger/exercises/api/views.py

```diff
@@ -221,11 +221,22 @@ class ExerciseCommentViewSet(viewsets.ReadOnlyModelViewSet):
     """
     API endpoint for exercise comment objects
     """
-    queryset = ExerciseComment.objects.all()
     serializer_class = ExerciseCommentSerializer
+
     ordering_fields = '__all__'
-    filterset_fields = ('comment',
-                        'exercise')
+    filterset_fields = ('exercise',
+                        'comment'
+                        )
```

Comment:
Why change the order here? :)

Reply:
Sorry! Didnt realise I did that. Fixed it now